Makefile

```diff
@@ -27,9 +29,15 @@ GO_TAGS=-tags codegen
 all: test
 
 build-ack-generate:	## Build ack-generate binary
-	@echo -n "building ack-generate ... "
-	@go build ${GO_TAGS} ${GO_LDFLAGS} -o bin/ack-generate cmd/ack-generate/main.go
-	@echo "ok."
+	@if [ ! -d "$(ACK_CODE_GENERATOR_SOURCE_PATH)" ]; then \
+		echo "Unable to find aws-controllers-k8s/code-generator source repository. Please git clone the aws-controllers-k8s/code-generator repository into your Go src path."; \
+	else \
+		echo -n "building ack-generate in aws-controllers-k8s/code-generator repo ... "; \
+		pushd "$(ACK_CODE_GENERATOR_SOURCE_PATH)" 1>/dev/null; \
+		go build $(GO_TAGS) $(GO_LDFLAGS) -o bin/ack-generate cmd/ack-generate/main.go; \
+		popd 1>/dev/null; \
+		echo "ok."; \
+	fi
```

Comment:
@mbaijal the above change in the Makefile works for me to build the ack-generate binary from the new aws-controllers-k8s/code-generator repository. Were you using this Makefile?

Reply:
Approving since I don't want to block merge and testing will be easier after.